### DISCUSSION TITLE
Ferium.exe wont run

### DISCUSSION THREAD

**Discussion by Izackadacka** [gorilla-devs/ferium#248] [Q&A] [answered]
I have no clue what I'm doing, obvious idiot. I installed ferium through rust cargo, and it says (executable `ferium.exe`) but when i go to the run the ferium.exe it doesn't open. Does anyone have any idea why it won't run?

**Comment by theRookieCoder** [comment, marked as answer]
Well ferium isn't an application. It runs through the CLI, so you have to use ferium through command prompt or powershell.

**Reply by Izackadacka** [reply]
How do you do that? kinder knew to all this.

**Reply by theRookieCoder** [reply]
Well my recommendation is that you stick to GUI applications (like PrismMC), because CLIs are harder to use especially for beginners. Anyways, you can open command prompt (search for it) and type in `ferium` to see the help page. The [readme](https://github.com/gorilla-devs/ferium#overview--help-page) is basically a tutorial so you can try using that to find your way around.

**Reply by Izackadacka** [reply]
Okay, thank you mate.
